### PR TITLE
gazebo_gps_plugin fix and enhancement

### PR DESCRIPTION
**Pull request by spurnvoj**
- fixed loading of 'topic' parameter from SDF
- renamed class member variables to the consistent coding convention (Google C++ style guide)
- fixed crashing of the plugin due to reading from shared variables

### CODE REVIEW

File: include/gazebo_gps_plugin.h [hunk 1]
```diff
@@ -71,66 +70,68 @@ class GAZEBO_VISIBLE GpsPlugin : public SensorPlugin
...
+  static constexpr const double   gps_corellation_time_ = 60.0;  // s
+  double                          gps_xy_random_walk_;
+  double                          gps_z_random_walk_;
+  double                          gps_xy_noise_density_;
+  double                          gps_z_noise_density_;
+  double                          gps_vxy_noise_density_;
+  double                          gps_vz_noise_density_;
+};  // class GAZEBO_VISIBLE GpsPlugin
+}  // namespace gazebo
+#endif  // _GAZEBO_GPS_PLUGIN_HH_
```
Comment (Jaeyoung-Lim) [inline include/gazebo_gps_plugin.h]: I know that we don't have style checks in this repository, but can you remove arbitrary white space fixes? At some point we need to introduce style fixes in this repo, but this shouldn't be done in a file to file basis.

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
Thanks for the contribution and the fix!

It would be better if we can remove the white spaces fixes

File: include/gazebo_gps_plugin.h (hunk 1 above)
Comment (spurnvoj) [reply include/gazebo_gps_plugin.h]: I probably called auto-indent and style fix in the header file. I will revert it. The cpp file should be fine.

**Review by Jaeyoung-Lim — APPROVED** [review]
Thanks!

**Pull request merged**